### PR TITLE
Removes custom demo and updates Grunt build.

### PR DESCRIPTION
**Pull request by himedlooff**

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -44,7 +44,19 @@ module.exports = function(grunt) {
 
       // Set whether or not to include html5shiv for demoing a component.
       // Only necessary if component patterns include new HTML5 elements
-      html5Shiv: false,
+      // html5Shiv: false,
```
Comment (Scotchester) [inline Gruntfile.js]: I think for the boolean values, we should leave them uncommented, or comment it out with it set to `true`, so the user only has to do one thing to turn it on, either uncomment or change `false` to `true`.

Comment (himedlooff) [inline Gruntfile.js]: Sounds good, done!

**Comment by Scotchester** [comment]
:+1:

**Comment by himedlooff** [comment]
Thanks yet again! :smiley_cat:
I see mucho code reviews in my future :crystal_ball:.

**Pull request merged**